src/test/java/com/shopify/ShopifySdkDriver.java

```diff
@@ -664,6 +665,19 @@ public void givenSomeValuesWhenUpdatingACustomerThenExpectValuesToBeUpdatedOnCus
 		assertEquals("RyanTest", updatedCustomer.getFirstName());
 	}
 
+	@Test
+	public void givenSomeErrorOccurrsWhenCreatingFulfillmentThenExpectCorrectErrors() {
+		try {
+			shopifySdk.createFulfillment(ShopifyFulfillmentCreationRequest.newBuilder().withOrderId("2854620102717")
+					.withTrackingCompany("UPS").withTrackingNumber("ABC-123").withNotifyCustomer(false)
+					.withLineItems(new LinkedList<>()).withLocationId("5523767400")
+					.withTrackingUrls(Arrays.asList("http://google.com/123")).build());
+		} catch (final Exception e) {
+			e.printStackTrace();
```

Comment:
Where is the assertion of this test?

Reply:
@ShannonTalarico this is a driver test, so it's just for testing the API. The fix for this is just testing the errors we are logging. Assertions are essentially meaningless here. I can remove the test though since it is not providing much value.

Reply:
@ryankazokas Thank you for clarifying.